README.md

```diff
@@ -163,26 +163,26 @@ go run main.go -metamorph=true
-| 109 | `REJECTED`             | The transaction has been rejected by the Bitcoin network.                                                                                                                                                |
-
-This status is returned in the `txStatus` field whenever the transaction is queried.
+| 10  | `QUEUED`               | The transaction has been queued for processing.                                                                                                                                                          |
+| 20  | `RECEIVED`             | The transaction has been properly received by the metamorph processor.                                                                                                                                   |
+| 30  | `STORED`               | The transaction has been stored in the metamorph store. This should ensure the transaction will be processed and retried if not picked up immediately by a mining node.                                  |
+| 40  | `ANNOUNCED_TO_NETWORK` | The transaction has been announced (INV message) to the Bitcoin network.                                                                                                                                 |
+| 50  | `REQUESTED_BY_NETWORK` | The transaction has been requested from metamorph by a Bitcoin node.                                                                                                                                     |
+| 60  | `SENT_TO_NETWORK`      | The transaction has been sent to at least 1 Bitcoin node.                                                                                                                                                |
+| 70  | `ACCEPTED_BY_NETWORK`  | The transaction has been accepted by a connected Bitcoin node on the ZMQ interface. If metamorph is not connected to ZMQ, this status will never by set.                                                 |
+| 80  | `SEEN_IN_ORPHAN_MEMPOOL`             | The transaction has been sent to at least 1 Bitcoin node but parent transaction was not found. |
+| 90  | `DOUBLE_SPEND_ATTEMPTED`             | The transaction is a double spend attempt. Competing transaction(s) will be returned with this status. |
+| 100   | `SEEN_ON_NETWORK`      | The transaction has been seen on the Bitcoin network and propagated to other nodes. This status is set when metamorph receives an INV message for the transaction from another node than it was sent to. |
+| 110  | `MINED`                | The transaction has been mined into a block by a mining node.                                                                                                                                            |
```

Comment:
Maybe it would be a good idea to switch the `MINED` and `REJECTED` statuses.

If the `MINED` status were the final one (with the highest value), we could remove the `AND metamorph.transactions.status != MINED` condition from the `WHERE` clauses in update queries. In my opinion, these conditions are not intuitive and it is easy to forget about them if someone new writes a query.

Reply:
You have a point. `MINED` should be more important than `REJECTED`. If one node sends us `MINED` but another node rejects our transaction for some reason, we don't want to update that status and keep it on `MINED`.

I applied this switch and simplified the logic as you suggested. Thanks.

@boecklim take a look at this and verify :)